Loop/View Models/AdverseEventReportViewModel.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  AdverseEventReportViewModel.swift
+//  Loop
+//
+//  Created by Nathaniel Hamming on 2020-10-02.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import LoopKit
+
+public class AdverseEventReportViewModel {
+    private let pumpStatus: PumpManagerStatus?
+
+    private let cgmDevice: HKDevice?
+
+    let formURLString = "https://support.tidepool.org/hc/en-us/requests/new?ticket_form_id=360000551951"
```

Comment:
I presume this URL (and the parameters) are valid?

Reply:
Yes. Nick and I have been testing this and he is happy with the results.